node/src/chain.rs

```diff
@@ -49,6 +52,8 @@ const TOPICS: &[u8] = &[
     Topics::Agreement as u8,
 ];
 
+const ACCEPT_BLOCK_TIMEOUT_SEC: u8 = 20;
```

Comment:
This can be a Duration instead of seconds. This way you don't need the conversion when using it. 

Furthermore, when this will be moved to configuration, users can set it up using their own format without struggling with conversion like:
https://github.com/dusk-network/kadcast/blob/1fd4d87e5666cd002a9756d19b90d7200a08e296/src/config.rs#L111-L112